gnome-initial-setup/gnome-initial-setup.c

```diff
@@ -286,6 +287,25 @@ get_mode (void)
     return GIS_DRIVER_MODE_NEW_USER;
 }
 
+static gboolean
+connect_to_gdm (void)
+{
+  GdmClient *client;
+  GdmGreeter *greeter;
+  GError *error = NULL;
```

Comment:
```suggestion
  g_autoptr(GError) error = NULL;
```

and remove the `g_error_free` call